test/integration/helpers/serverspec/spec_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'busser/rubygems'
+Busser::RubyGems.install_gem('beaneater', '~> 0.3.3')
```

Comment:
Another option would have been to put a `Gemfile` in the `test/integration/helpers/serverspec` and serverspec will bundle for you.